crates/nu-utils/src/default_files/sample_config.nu

```diff
@@ -912,18 +914,18 @@ const NU_PLUGIN_DIRS = $NU_PLUGIN_DIRS ++ [($nu.default-config-dir | path join '
 
 # Appending to the OS path is a common configuration task.
 # Because of the previous ENV_CONVERSIONS (performed internally
-# before your config.nu loads), the path variable is a list that can 
+# before your config.nu loads), the path variable is a list that can
 # be appended to using, for example:
-$env.path ++= "~/.local/bin"
+$env.PATH ++ [ "~/.local/bin" ]
```

Comment:
I am unsure why this raises an error, while https://github.com/nushell/nushell/blob/39770d41975a0dae3cdc89802215416a4c6671e4/crates/nu-utils/src/default_files/sample_config.nu#L420 doesn't

Reply:
It's the recent change from `++` acting as an `append` to having it be a concatenate.  Before, the operand could be any value; now it must be a list.